io.github.strozer2002.sparrow-linux.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.strozer2002.sparrow-linux
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R bin/ /app/sparrow
+      - chmod +x /app/sparrow/sparrow
+      - mkdir /app/bin/
+      - mkdir /app/lib/
+      - ln -s /app/sparrow/sparrow /app/bin/sparrow
+      - install -Dm644 io.github.strozer2002.sparrow-linux.metainfo.xml /app/share/appdata/io.github.strozer2002.sparrow-linux.metainfo.appdata.xml
```

Comment:
```suggestion
      - install -Dm644 io.github.strozer2002.sparrow-linux.metainfo.xml /app/share/metainfo/io.github.strozer2002.sparrow-linux.metainfo.xml
```

Reply:
resolved

Reply:
> resolved

It's still `metainfo.appdata.xml`, the extension is either `appdata.xml` or `metainfo.xml`

Reply:
i resolved this, check another one please